velox/exec/tests/OrderByTest.cpp

```diff
@@ -1346,3 +1348,63 @@ DEBUG_ONLY_TEST_F(OrderByTest, spillWithNoMoreOutput) {
   ASSERT_EQ(planStats.spilledRows, 0);
   OperatorTestBase::deleteTaskAndCheckSpillDirectory(task);
 }
+
+TEST_F(OrderByTest, maxSpillBytes) {
+  const auto rowType =
+      ROW({"c0", "c1", "c2"}, {INTEGER(), INTEGER(), VARCHAR()});
+  const auto vectors = createVectors(rowType, 1024, 15 << 20);
+  auto planNodeIdGenerator = std::make_shared<core::PlanNodeIdGenerator>();
+  core::PlanNodeId orderNodeId;
+  const auto plan =
+      PlanBuilder(planNodeIdGenerator)
+          .values(vectors)
+          .orderBy({fmt::format("{} ASC NULLS LAST", "c0")}, false)
+          .capturePlanNodeId(orderNodeId)
+          .planNode();
+  const auto expectedResult = AssertQueryBuilder(plan).copyResults(pool_.get());
+
+  auto spillDirectory = exec::test::TempDirectoryPath::create();
+  auto queryCtx = std::make_shared<core::QueryCtx>(executor_.get());
+
+  // Default maxSpillBytes is greater than the input size.
+  AssertQueryBuilder(plan)
+      .spillDirectory(spillDirectory->path)
+      .queryCtx(queryCtx)
+      .config(core::QueryConfig::kSpillEnabled, "true")
+      .config(core::QueryConfig::kOrderBySpillEnabled, "true")
+      // Set a small capacity to trigger threshold based spilling
+      .config(
+          QueryConfig::kOrderBySpillMemoryThreshold, std::to_string(5 << 20))
+      .copyResults(pool_.get());
+
+  auto testSpill = [&](int32_t maxSpilledBytes) {
+    AssertQueryBuilder(plan)
```

Comment:
Since we use struct {} testSettings to structure the test, then we don't need testSpill callback. Or you don't need testSettings structure. Thanks!